src/VAP/LineDetector/DetectionLine.cs

```diff
@@ -15,11 +15,33 @@ public class DetectionLine
 
         public static double DEFAULT_OCCUPANCY_THRESHOLD = 0.9; // default threhsold
 
-        public Point p1;
-        public Point p2;
-        public double increment;
-        public double overlapFractionThreshold = DEFAULT_OCCUPANCY_THRESHOLD;
-
+        /// <summary>
+        /// The first point of the line.
+        /// </summary>
+        public Point p1 { get; set; }
```

Comment:
❔ While the fields were not marked `readonly`, it's not clear that was intentional. Is it possible to make these properties readonly?

```suggestion
        public Point p1 { get; }
```